.github/workflows/tests.yml

```diff
@@ -60,7 +60,7 @@ jobs:
            composer update --prefer-dist --no-interaction --no-progress --${{ matrix.stability }}
 
       - name: Execute tests
-        run: vendor/bin/pest
+        run: vendor/bin/pest -vvv
```

Comment:
Pest truncates output a lot for failed assertions.

This will allow us to actually see why a test failed.